lesson07/assignment/index.html/App.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div id="app">
+<ul>
+<li v-for="link in links"> {{link}}</li>
+</ul>
+<div><button @click="awesome = !awesome">Click Me!</button>
+    <h1 v-if="awesome">Plant Portraits are Awesome!</h1>
+    <h1 v-else>Don't hesitate, order one now!</h1>>Plant Portraits</h1>
```

Comment:
Invalid HTML. Should be `<h1 v-else>Don't hesitate, order one now! Plant Portraits</h1>`